internal/controllers/port/status.go

```diff
@@ -35,9 +39,8 @@ func (portStatusWriter) GetApplyConfigConstructor() generic.ORCApplyConfigConstr
 	return orcapplyconfigv1alpha1.Port
 }
 
-func (portStatusWriter) ResourceIsAvailable(_ orcObjectPT, osResource *osResourceT) bool {
-	// A port is available as soon as it exists
-	return osResource != nil
+func (portStatusWriter) ResourceIsAvailable(orcObject orcObjectPT, osResource *osResourceT) bool {
+	return orcObject.Status.ID != nil && osResource != nil && osResource.Status == PortStatusActive
```

Comment:
Actually, both ACTIVE _and_ DOWN are correct status for availability.

The other two statuses, BUILD and ERROR, are effectively not available.